en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/01-adding-pages-to-sites-intro.markdown

```diff
@@ -12,3 +12,52 @@ fact, sites primarily exist for the sake of organizing pages and content, so
 now you'll learn about the different types of pages in Liferay, and how to 
 select the best tools based on your use cases. You'll also learn how to manage 
 pages and use various configuration options.
+
+Before diving into page creation, you should be informed on the two major page
+types provided in @product-ver@:
+
+[*Content Pages*](/docs/7-2/user/-/knowledge_base/u/creating-content-pages): the
+new page type providing more flexibility, especially for non-technical users.
+
+[*Widget Pages*](/docs/7-2/user/-/knowledge_base/u/adding-applications-to-a-page):
+the traditional page type that is more rigid. There may still be some use cases
+where it's preferred.
+
+You should always opt for Content Pages, unless there's a concrete reason
+otherwise. Content Pages are being actively developed and offer many key
```

Comment:
Reading "are being actively developed" makes me think that other features are not, and potentially also that they are still half baked. I wonder if there is some other way of wording this that just praises Content Pages but doesn't leave this taste.